test/test_util.dart

```diff
@@ -7,10 +7,10 @@ import 'dart:async';
 import 'package:shelf/shelf.dart';
 
 // "hello,"
-const HELLO_BYTES = const [104, 101, 108, 108, 111, 44];
+const helloBytes = const [104, 101, 108, 108, 111, 44];
```

Comment:
Curious, has the style guide recommendation changed? I know flutter uses `kMyConstantName`; I thought regular dart was still `MY_CONSTANT_NAME`.